Update using ember-cli-update

mansona opened the pull request:
I opened this PR to essentially test ember canary and beta because they seem to fail now with an error: `You attempted to overwrite the built-in helper "array" which is not allowed. Please rename the helper.`

I don't know why ember <= 3.4 is failing with the error `array is not a helper` 🤔 but we might want to drop some versions that we support if we were to update like this. Thoughts?

I'm going to work on a different PR that addresses https://github.com/kellyselden/ember-array-helper/issues/89 now 👍

kellyselden commented:
@mansona Do you think you will be picking this back up?

mansona commented:
@kellyselden I can pick it back up, but we need a decision on what to do about the versions <= 3.4 🤔 

I don't  understand the error and really we should discuss maybe dropping them? what are your thoughts?

kellyselden commented:
Have you tried it without the new syntax in the dummy app?

mansona commented:
What do you mean by "without the new syntax"? Do you mean without angle brackets? I remember trying a few things but I couldn't understand why the helper just wasn't showing up 🤔 did something change about the way that you register helpers after 3.4?

kellyselden commented:
> Do you mean without angle brackets?

Yes. That was my guess as to why the tests were failing in older versions.

mansona commented:
@kellyselden so I've updated the PR so that it's no longer conflicting with master, and now I've changed nothing with the syntax and just fixed the linting errors by turning off the rules 😂 

This is still showing the same error with Ember <= 3.4 🤔 `array is not a helper` which I just don't understand 🤔

kellyselden commented:
I wouldn't worry about updating it then. Unless you want to increment in chunks to figure it out, I'd say close it.

mansona commented:
let's close it for now and we can come back around later if we felt like it :)
